image/helpers.go

```diff
@@ -417,6 +417,12 @@ var writeResolvedContent = writeResolvedContentImpl
 // E.g.
 // /tmp/prep-img/resolved-content/pi/part0/{config.txt,bootcode.bin,...}
 func writeResolvedContentImpl(prepareDir string, info *gadget.Info, gadgetUnpackDir, kernelUnpackDir string) error {
+	// double check that things make sense
+	// XXX: move into setupSeed() instead?
```

Comment:
yes, I think one level up in setupSeed would be a clearer placement  after the ReadInfoAndValidate